### ISSUE TITLE
Great expectations suite creation takes too long

### ISSUE THREAD

**Issue by armaandhull** [fivetran/great_expectations#1670]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when `great_expectations suite new` takes a very long time scanning the entire database which seems quite unnecessary given that I already know which schema and table I intend to create the suite from. 

On top of that, even after scanning the entire database for tables and schemas, the cli only shows the first 5 (alphabetically sorted) tables in the 1st schema as options to choose from. IMO the odds of a user picking from the first 5 tables exposed by the CLI is quite low and it also slows down the expectation suite creation.

**Describe the solution you'd like**
Ideally `great_expectations suite new` should first ask for a schema qualified table name (or alternatively a SQL query) and then scan the `information_schema` only for the relevant table's properties.

**Additional context**
We run Great Expectations in production against Snowflake and see the following query getting executed for every single schema in the database as the first step in the flow.

```
SELECT /* sqlalchemy:_get_columns_in_schema */
               ic.table_name,
               ic.column_name,
               ic.data_type,
               ic.character_maximum_length,
               ic.numeric_precision,
               ic.numeric_scale,
               ic.is_nullable,
               ic.column_default,
               ic.is_identity,
               ic.comment
          FROM information_schema.columns ic
         WHERE ic.table_schema='<schema_name>'
         ORDER BY ic.ordinal_position
```

**Comment by spbail** [comment]
@eugmandel This is the other issue I mentioned that came out of a user interview.

**Comment by Shinnnyshinshin** [comment]
addressed as part of the #2244
